iroha-balancer-client/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.gradle.api.tasks.bundling.Jar
+import org.springframework.boot.gradle.tasks.bundling.BootJar
+
+plugins {
+    `java-library`
+    kotlin("jvm") // version "1.3.72"
+}
+
+val camelVersion = "2.23.2"
+val irohaJavaVersion = "6.2.0"
+
+dependencies {
+    // Kotlin
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+
+    // Logging
+    api("io.github.microutils:kotlin-logging:1.7.9")
+
+    // Apache Commons
+    implementation("org.apache.commons:commons-collections4:4.4")
+
+    // RabbitMQ
+    implementation("com.rabbitmq:amqp-client:5.9.0")
+
+    // testcontainers
+    testCompile("org.testcontainers:testcontainers:1.14.3")
+    testImplementation("org.testcontainers:junit-jupiter:1.14.3")
+
+    // Jackson
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.11.+")
+
+    // Iroha java library
+    implementation("com.github.hyperledger.iroha-java:client:${irohaJavaVersion}")
+    testImplementation("com.github.hyperledger.iroha-java:client:${irohaJavaVersion}")
```

Comment:
it seems we can remove `testImplementation` one

Reply:
Sure